bin/common.sh

```diff
@@ -0,0 +1,196 @@
+manifests_dir="installation/manifests/product"
+default_manifests="${manifests_dir}/000-subscription-serverless.yaml,${manifests_dir}/000-subscription-hyperfoil.yaml,${manifests_dir}/000-subscription-amq-streams.yaml,${manifests_dir}/100-kafka.yaml,${manifests_dir}/100-hyperfoil.yaml,${manifests_dir}/100-knative-eventing.yaml,${manifests_dir}/100-knative-kafka.yaml"
+
+cluster_domain="$(oc get ingresses.config/cluster -o jsonpath='{.spec.domain}')"
+default_hyperfoil_server_url="hyperfoil-cluster-hyperfoil.${cluster_domain}"
+export HYPERFOIL_SERVER_URL=${HYPERFOIL_SERVER_URL:-${default_hyperfoil_server_url}}
+export KNATIVE_MANIFESTS=${KNATIVE_MANIFESTS-$default_manifests}
+export SKIP_DELETE_RESOURCES=${SKIP_DELETE_RESOURCES:-false}
+export TEST_CASE_NAMESPACE=${TEST_CASE_NAMESPACE-"perf-test"}
+export WORKER_ONE=${WORKER_ONE:-node-role.kubernetes.io/worker=""}
+
+alias kubectl=oc
+
+function create_namespaces {
+  echo "Creating namespaces"
+  oc create ns kafka --dry-run=client -oyaml | oc apply -f - || return $?
+}
+
+function delete_test_namespace {
+  if [ -z "${TEST_CASE_NAMESPACE}" ]; then
+    oc delete ns "${TEST_CASE_NAMESPACE}" --ignore-not-found --timeout=10m || return $?
+  fi
+}
+
+function delete_namespaces {
+  # TODO(pierDipi) run must-gather(s) or backup prometheus metrics for further analysis.
+  echo "Deleting namespaces"
+  delete_test_namespace || return $?
+  oc delete ns kafka --ignore-not-found --timeout=30m || return $?
+  oc delete ns knative-eventing --ignore-not-found --timeout=30m || return $?
+}
+
+function apply_manifests() {
+  create_namespaces || return $?
+
+  # Extract manifests from the comma-separated list of manifests
+  IFS=\, read -ra manifests <<<"${KNATIVE_MANIFESTS}"
+
+  for x in "${manifests[@]}"; do
+    echo "Applying ${x}"
+    envsubst <"${x}" | oc apply -f - || return $?
+    wait_for_operators_to_be_running || return $?
```

Comment:
```suggestion
    envsubst <"${x}" | oc apply -f - || return $?
    sleep 30
    wait_for_operators_to_be_running || return $?
```

Reply:
Due to https://github.com/openshift-knative/eventing-hyperfoil-benchmark/issues/29#issuecomment-1099408603